cli/cmds/cluster_create_flags.go

```diff
@@ -60,6 +61,18 @@ func NewCreateFlags(config *CreateConfig) []cli.Flag {
 			Usage:       "storage class name for dynamic persistence type",
 			Destination: &config.storageClassName,
 		},
+		&cli.StringFlag{
+			Name:        "storage-request-size",
+			Usage:       "storage size for dynamic persistence type",
+			Destination: &config.storageRequestSize,
+			Action: func(ctx *cli.Context, value string) error {
+				_, err := resource.ParseQuantity(value)
```

Comment:
sinec we dont need the value we can wrap the error
```suggestion
				if _, err := resource.ParseQuantity(value); err != nil {
				  return errors.New(`invalid storage size, should be a valid resource quantity e.g "10Gi"`) 
				}
```